tests/gui/ertwidgets/models/test_analysismodulevariablesmodel.py

```diff
@@ -0,0 +1,15 @@
+from ert_gui.ertwidgets.models.analysismodulevariablesmodel import (
+    AnalysisModuleVariablesModel,
+)
+
+
+QT_MAX_SIGNED_INT = 2147483647
+
+
+def test_getVariableMaximumValue():
+    for entry in AnalysisModuleVariablesModel._VARIABLE_NAMES.items():
+        if "max" in entry[1] and entry[0] != "CV_NFOLDS":
+            assert (
+                AnalysisModuleVariablesModel.getVariableMaximumValue(entry[0])
```

Comment:
Is it perhaps better to just run it and see that it does not crash? Also,  do we need to check if there is a "max", and that it is not "CV_NFOLDS"?

Reply:
Just an idea. is it possible to initialize the spinner thingy, and thus check that it does not crash that way? I guess all this code is simply for initializing the spinner?